File: app/src/main/java/com/expedia/assignment/detail/model/Response.java
package com.expedia.assignment.detail.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Response {

    @SerializedName("photos")
    @Expose
    private Photos photos;

    public Photos getPhotos() {
        return photos;
    }

    public void setPhotos(Photos photos) {
        this.photos = photos;
    }

}

File: app/src/main/java/com/expedia/assignment/search/model/Response.kt
package com.expedia.assignment.search.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Response {

    @SerializedName("venues")
    @Expose
    var venues: List<Venue>? = null
    @SerializedName("geocode")
    @Expose
    var geocode: Geocode? = null

}
